projects/demo/src/modules/experimental/badge/badge.template.html

```diff
@@ -0,0 +1,164 @@
+        </tui-doc-example>
+
+        <tui-doc-example
+            id="custom"
+            heading="Customization"
+            [content]="example5"
+        >
+            <tui-badge-example-5></tui-badge-example-5>
+        </tui-doc-example>
+    </ng-template>
+
+    <ng-template pageTab>
+        <tui-doc-demo>
+            <p>
+                <tui-badge
+                    *ngIf="contentType === 'icon'"
+                    [appearance]="appearance"
+                    [hoverable]="hoverable"
+                    [size]="size"
+                    [value]="values[value]"
+                >
+                    <tui-svg src="tuiIconCheck"></tui-svg>
+                </tui-badge>
+
+                <tui-badge
+                    *ngIf="contentType === 'empty'"
+                    [appearance]="appearance"
+                    [hoverable]="hoverable"
+                    [size]="size"
+                    [value]="values[value]"
+                ></tui-badge>
+
+                <tui-badge
+                    *ngIf="contentType === 'image'"
+                    [appearance]="appearance"
+                    [hoverable]="hoverable"
+                    [size]="size"
+                >
+                    <img
+                        alt="market"
+                        src="assets/images/market.png"
+                    />
+                </tui-badge>
+            </p>
+        </tui-doc-demo>
+        <label
+            tuiLabel="Content type"
+            class="tui-space_vertical-3"
+        >
+            <tui-radio-list
+                orientation="horizontal"
+                [items]="contentTypeVariants"
+                [(ngModel)]="contentType"
+            ></tui-radio-list>
+        </label>
+        <tui-doc-documentation>
+            <ng-template
+                documentationPropertyMode="input"
+                documentationPropertyName="size"
+                documentationPropertyType="TuiSizeL"
+                [documentationPropertyValues]="sizeVariants"
+                [(documentationPropertyValue)]="size"
+            >
+                Size
+            </ng-template>
+            <ng-template
+                documentationPropertyMode="input"
+                documentationPropertyName="appearance"
+                documentationPropertyType="TuiStatus"
+                [documentationPropertyValues]="appearanceVariants"
+                [(documentationPropertyValue)]="appearance"
+            >
+                Appearance
+            </ng-template>
+            <ng-template
+                documentationPropertyMode="input"
+                documentationPropertyName="value"
+                documentationPropertyType="string | number"
+                [documentationPropertyValues]="valueVariants"
+                [(documentationPropertyValue)]="value"
+            >
+                Value. If value is a number and is more than 99, it will be replaced with &laquo;99+&raquo;
```

Comment:
Can we use `value: number` with icon?

Is it expected behaviour (no horizontal padding around) ?
<img width="269" alt="Screenshot 2023-09-01 at 11 04 50" src="https://github.com/taiga-family/taiga-ui/assets/35179038/a497fb47-e12f-42cb-9a46-f9dd3c0f9910">